Manatee.Json/Pointer/JsonPointer.cs

```diff
@@ -101,7 +115,7 @@ public override string ToString()
 		/// </summary>
 		public JsonPointer Clone()
 		{
-			return new JsonPointer(this){_usesHash = _usesHash};
+			return new JsonPointer(this, _usesHash);
```

Comment:
Unless we're making `_usesHash` read only, I don't see the point in this.

Reply:
The point here is to avoid using the public constructor and executing line 40 to set _useHash when the value is gonna be set by the initializer in the Clone method anyway